src/constants/chainInfo.tsx

```diff
@@ -68,7 +68,6 @@ export const ChainInfo: ChainInfo = {
     chainId: ChainId.Sepolia,
     networkName: "sepolia",
     explorerUrl: "https://sepolia.etherscan.io",
-    rpcUrl: "https://rpc.sepolia.org",
     nativeCurrency: {
```

Comment:
this will fallback to infuraProvider, seemed to work. due to urgency, let's try this first